docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,387 @@
+# Getting Started
+This document is intended as a "getting started" guide. The audience are developers looking to modify or *publish* the packages to the Universe. This document is written in tutorial and walk-through format. The goal is to help you "get started". It does not go into great depth on some of the conceptual or inner details. This guide aims at making a user familiar with the concepts of what a Package is and what are the roles of Marathon and Universe in the package life cycle.
+
+#### Table of Contents
+- [Getting Started](#getting-started)
```

Comment:
This bullet isn't really needed, since we're basically at the top of the document already. That will remove one level of nesting of the TOC.